packages/lighthouse-config/src/index.ts

```diff
@@ -3,14 +3,39 @@ interface Params {
   urls: string[]
 }
 
+/**
+ * Some values, we took from the default lighthouse config
+ * throttling mobileSlow4G: https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/constants.js#L22
+ */
+
 const lhConfig = ({ urls, server }: Params) => {
   const url = urls.map((path) => `${server}${path}`)
 
   return {
     ci: {
       collect: {
         url,
-        numberOfRuns: 5,
+        settings: {
```

Comment:
Aren't these the default config already ? 

Also, we should figure out a way to run two different devices, one using desktop and another using mobile

Reply:
for it, you should use the extends config, like this:

```
// lhci config
  extends: 'lighthouse:default',
``

about run both devices, still is not possible, you can see more about [here](https://github.com/GoogleChrome/lighthouse-ci/issues/138).

Reply:
Cool ! what about http1 ?

Reply:
eita, I had forgotten about it, I'll see what I can do.

Reply:
done